valhalla/filesystem.h

```diff
@@ -359,4 +359,4 @@ inline bool remove_all(const path& p) {
   return remove(p);
 }
 
-} // namespace filesystem
```

Comment:
@kevinkreiser Where did this come from?

Reply:
looks like i deleted the newline. id ont know man it was a  big hairy diff!